ppa/archive/views.py

```diff
@@ -64,17 +65,35 @@ def get_queryset(self, **kwargs):
             # to support exact phrase searches
             solr_q = 'text:(%s) OR {!join from=srcid to=id v=$join_query}' % query
             # sort by relevance, return score for display
-            self.sort = 'relevance'
+            self.sort = 'Relevance'
```

Comment:
After testing it, I think you need to move the default sort logic into the form class because otherwise on the display the current sort option isn't selected.